server/svix-server/src/queue/mod.rs

```diff
@@ -40,11 +40,67 @@ impl MessageTask {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MessageTaskBatch {
+    pub msg_id: MessageId,
+    pub app_id: ApplicationId,
+    pub endpoint_ids: Vec<EndpointId>,
+    pub trigger_type: MessageAttemptTriggerType,
+    pub attempt_count: u16,
```

Comment:
Almost all of these are not needed.
- msg_id is needed
- app_id is needed I guess for fetching msg_id, but otherwise not really needed.
- endpoint_ids are not needed, you can just fetch the CreateMessageApp and get them back.
- attempt_count is always 0 for batch, so not needed.